### PR TITLE
Bugfix wrong list of forced payments used in the payment use case

### PR DESCRIPTION
**Pull request by rwrzesien**
Resolves https://github.com/golemfactory/concent/issues/982

I have also added tests for `sci_backend` because there were none so far for such important module.

### CODE REVIEW

File: concent_api/core/tests/test_payments_sci_backend.py [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+            )
+
+        self.assertEqual(deposit_value, self.deposit_value)
+
+        new_sci_rpc.return_value.get_deposit_value.assert_called_with(
+            self.task_to_compute.requestor_ethereum_address,
+        )
+
+    def test_that_sci_backend_force_subtask_payment_should_return_transaction_hash(self):
+        PaymentInterface.__instance = None
```
Comment (dybi) [inline concent_api/core/tests/test_payments_sci_backend.py]: Is it needed? If so, why?

Comment (rwrzesien) [reply concent_api/core/tests/test_payments_sci_backend.py]: No, it is leftover for my previous, not-working solution for mocking singleton object. It seems that if I have mocked it once during the test, then in other tests I couldn't overwrite it. I ended up mocking the whole `__new__` function instead of inner function which was called only if the instance does not already exist.
